Shared/Samples/Update related features/UpdateRelatedFeaturesView.swift

```diff
@@ -0,0 +1,294 @@
+                    }
+                }
+            // Load initial map and data when the view appears.
+                .task {
+                    isLoading = true
+                    defer { isLoading = false }
+                    do {
+                        try await model.loadFeatures()
+                        // Set initial viewpoint to Alaska.
+                        await mapView.setViewpoint(
+                            Viewpoint(latitude: 65.399121, longitude: -151.521682, scale: 50000000)
+                        )
+                    } catch {
+                        self.error = error
+                    }
+                }
+            // Show a loading spinner when `isLoading` is true.
+                .overlay(alignment: .center) {
+                    if isLoading {
+                        loadingView
+                    }
+                }
+            // Display error alerts, if needed.
+                .errorAlert(presentingError: $error)
+        }
+    }
+    
+    /// A view displaying callout content, including editable "Annual Visitors" values.
+    /// Includes a picker to allow updating the selected visitor range.
+    var calloutContent: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            Text("\(model.parkName)")
+                .font(.headline)
+            if !model.attributeValue.isEmpty {
+                Text("Annual Visitors:")
+                // Picker to allow the user to update visitor range.
+                Picker("Annual Visitors", selection: $model.selectedVisitorValue) {
+                    ForEach(model.visitorOptions, id: \.self) { option in
+                        Text(option).tag(option)
+                    }
+                }
+                .pickerStyle(.menu)
+                .onChange(of: model.selectedVisitorValue) { _, newValue in
+                    Task {
+                        do {
+                            try await self.model.setSelectedFeatureUpdate(newValue)
+                        } catch {
+                            self.error = error
+                        }
+                    }
+                }
+            }
+        }
+        .padding()
+    }
+    
+    /// The loading indicator overlay shown during data fetches.
+    var loadingView: some View {
+        ProgressView(
+               """
+               Fetching
+               data
+               """
+        )
+        .padding()
+        .background(.ultraThinMaterial)
+        .clipShape(.rect(cornerRadius: 10))
+        .shadow(radius: 50)
+        .multilineTextAlignment(.center)
+    }
+}
+
+extension UpdateRelatedFeaturesView {
+    @MainActor
+    @Observable
+    class Model {
+        /// A map with a topographic basemap style.
+        @ObservationIgnored var map = Map(basemapStyle: .arcGISTopographic)
+        
+        /// A boolean value the reflects whether the callout should be shown or not.
+        var calloutVisible = false
```

Comment:
```suggestion
        var calloutIsVisible = false
```
A Boolean name should read as an assertion.